src/handlers/payout/post.ts

```diff
@@ -119,7 +129,139 @@ export const incentivizeComments = async () => {
   logger.info(`Skipping to generate a permit url for missing accounts. fallback: ${JSON.stringify(fallbackReward)}`);
 
   await addCommentToIssue(comment, issue.number);
-};
+}
+
+export async function incentivizePullRequestReviews() {
+  const logger = getLogger();
+  const {
+    mode: { incentiveMode, paymentPermitMaxPrice },
+    price: { baseMultiplier, incentives },
+    payout: { paymentToken, rpc },
+  } = getBotConfig();
+  if (!incentiveMode) {
+    logger.info(`No incentive mode. skipping to process`);
+    // return;
+  }
+  const context = getBotContext();
+  const payload = context.payload as Payload;
+  const issue = payload.issue;
+  if (!issue) {
+    logger.info(`Incomplete payload. issue: ${issue}`);
+    return;
+  }
+
+  if (issue.state_reason !== StateReason.COMPLETED) {
+    logger.info("incentivizePullRequestReviews: comment incentives skipped because the issue was not closed as completed");
+    // return;
+  }
+
+  if (paymentPermitMaxPrice == 0 || !paymentPermitMaxPrice) {
+    logger.info(`incentivizePullRequestReviews: skipping to generate permit2 url, reason: { paymentPermitMaxPrice: ${paymentPermitMaxPrice}}`);
+    // return;
+  }
+
+  const issueDetailed = bountyInfo(issue);
+  if (!issueDetailed.isBounty) {
+    logger.info(`incentivizePullRequestReviews: its not a bounty`);
+    // return;
+  }
+
+  const pulls = await getAllPullRequests(context, "closed");
+  if (pulls.length === 0) {
+    logger.debug(`incentivizePullRequestReviews: No pull requests found at this time`);
+    return;
+  }
+
+  let linkedPull;
+
+  for (const pull of pulls) {
+    const pullRequestLinked = await gitLinkedIssueParser({
+      owner: payload.repository.owner.login,
+      repo: payload.repository.name,
+      issue_number: pull.number,
```

Comment:
renamed some variables names

> Also can you check if it's possible that we do that same thing but opposite way, so getting linked PRs from issue - this way we can avoid this for loop

for loop will be necessary seems there is no feature which enables us to get the linked issues of a pr directly.
So the only solution is just analyzing the PR's description and extract the issue number.
So, to get linked prs of an issue we need to go through all the PRs

Reply:
> renamed some variables names
> 
> > Also can you check if it's possible that we do that same thing but opposite way, so getting linked PRs from issue - this way we can avoid this for loop
> 
> for loop will be necessary seems there is no feature which enables us to get the linked pr of an issue directly. So the only solution is just analyzing the PR's description and extract the issue number. So, to get linked prs of an issue we need to go through all the PRs

- You should parse from the sidebar because not all repositories will use the proper "Resolves #1234" syntax 
- The sidebar includes linked pull requests both with this "Resolves #1234" syntax, as well as manually linking by using the GitHub sidebar.

Reply:
`gitLinkedIssueParser` is searching for linked issues given a PR number, you need to create a similar function `gitLinkedPrParser` but for searching linked PRs given an issue number.